src/App.jsx

```diff
@@ -0,0 +1,86 @@
+import "./styles.css";
+import { Link, Route, BrowserRouter as Router, Switch } from "react-router-dom";
+import { Cart, Product, Toast, WishList } from "./components";
+import { useAxios } from "./api";
+import { useEffect } from "react";
+import { useProduct } from "./contexts";
+import { Actions } from "./reducers";
+import { getFilteredList } from "./reducers";
+
+export default function App() {
+  const {
+    state: { wishList, cartList },
+    dispatch,
+  } = useProduct();
+  const [productList, loadingStatus] = useAxios("/api/products");
+  const [wishItems] = useAxios("api/wishes");
+  const [cartItems] = useAxios("api/carts");
+
+  useEffect(() => {
+    if (cartItems.carts) {
```

Comment:
This is again for my curiosity. Why are you only initializing based on `cartItems.cart` ?